students/khtruong/lesson_03/list_lab.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+
+"""Sample module docstrings text
+"""
+
+
+def fruit_list():
+    '''Return fruit list.'''
+    return ['Apples', 'Pears', 'Oranges', 'Peaches']
+
+
+def add_fruit():
+    '''Return user input for type of fruit to add.'''
+    return input('Enter a type of fruit to add to list> ')
+
+
+def remove_fruit(l):
+    '''Return user input for type of fruit to delete.'''
+    keepgoing = True
+    while keepgoing:
+        del_item = input('Enter a type of fruit to remove from list> ')
+        # looking for valid item...keeps asking user until do
+        for item in l:
+            if item == del_item:
+                keepgoing = False
+                break
+        else:
+            print('{} is not found...try again!'.format(del_item))
+
+        # actually delete the item in list for all occurrences
+        for item in l:
+            if item == del_item:
+                l.remove(item)
+
```

Comment:
You could use the `in` operator here to check list membership and simplify things a bit:

```
def remove_fruit(l):
    '''Return user input for type of fruit to delete.'''
    keepgoing = True
    while keepgoing:
        del_item = input('Enter a type of fruit to remove from list> ')
        # looking for valid item...keeps asking user until do
        if del_item in l:
            keepgoing = False
        else:
            print('{} is not found...try again!'.format(del_item))
    # actually delete the item in list for all occurrences
    while del_item in l:
        l.remove(del_item)
```